test/deadLetterProcessors.js

```diff
@@ -56,6 +56,30 @@ describe('Deadletter processors', function() {
     });
   });
 
+  // eslint-disable-next-line
```

Comment:
why is this needed?

Reply:
long story short, it's not needed. 

long story long: i thought it was jasmine so I was using fdescribe. little did I know it was mocha (also these tests are FAST)